### PR TITLE
Updating EntitySearch readme to include correct namespace and add-credentials flag

### PR DESCRIPTION
**Pull request by stefanvalianu**
Note, not sure what the appropriate way to include the --add-credentials flag when running generate.cmd in the SDK directory is. Apparently this readme config file expects key value pairs on params, so I threw in a dummy value of y to get the generation to work.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by dsgouda — COMMENTED** [review]
Please note that this could be a breaking change.

File: specification/cognitiveservices/data-plane/EntitySearch/readme.md [hunk 1]
```diff
@@ -18,13 +18,14 @@ These settings apply only when `--tag=release_1_0` is specified on the command l
 
 ``` yaml $(tag) == 'release_1_0'
 input-file: v1.0/EntitySearch.json
+add-credentials: y
```
Comment (dsgouda) [inline specification/cognitiveservices/data-plane/EntitySearch/readme.md]: A. `add-credentials` is a boolean so this should look like `add-credentials:true`
B. Is there a reason why this flag is being set only for `release_1_0` tag? Looks like a code generation setting which should be independent of the tags; i.e. should be defined under the `# Code Generation` settings
An example [here](https://raw.githubusercontent.com/Azure/azure-rest-api-specs/8e7f780de76fa4005ca34898440f7810fa377fa9/specification/recoveryservicessiterecovery/resource-manager)/readme.md

**Comment by stefanvalianu** [comment]
Our SDK is not published yet so whether or not this is a breaking change isn't super concerning to us. (The currently checked in SDK is based off of the spec in this PR)

File: specification/cognitiveservices/data-plane/EntitySearch/readme.md (hunk 1 above)
Comment (stefanvalianu) [reply specification/cognitiveservices/data-plane/EntitySearch/readme.md]: Sure thing(s)

**Review by dsgouda — APPROVED** [review]
LGTM, breaking change acknowledged

**Pull request merged**